<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>303</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checklatestversion">
         <property name="text">
          <string>Check for latest version on program start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkstartwithsystem">
         <property name="text">
          <string>Start with System</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="checkupdates">
         <property name="title">
          <string>Check for package updates on program start</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QCheckBox" name="ckdlPackage">
            <property name="text">
             <string>also download new and updated packages</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>On Closing window</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="checkAskClose">
            <property name="text">
             <string>Ask</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="checkCloseClsoe">
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="checkCloseMinimize">
            <property name="text">
             <string>Minimize to tray</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Software</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="saveDownloadedCheck">
         <property name="text">
          <string>Save downloaded Files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ckShowAllApps">
         <property name="text">
          <string>show all installed applications</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Install Mode</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QRadioButton" name="checkAskInstall">
              <property name="text">
               <string>Ask</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="checkSilentInstall">
              <property name="text">
               <string>Silent</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="checkAttendeInstall">
              <property name="text">
               <string>Attended</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Upgrade Mode</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QRadioButton" name="checkUpgradeAsk">
              <property name="text">
               <string>Ask</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="checkSilentUpgrade">
              <property name="text">
               <string>Silent</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="checkAttendedUpgrade">
              <property name="text">
               <string>Attended</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Uninstall Mode</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QRadioButton" name="checkUninstallAsk">
              <property name="text">
               <string>Ask</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="checkSilentUninstall">
              <property name="text">
               <string>Silent</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="checkAttendedUninstall">
              <property name="text">
               <string>Attended</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tasks</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QSpinBox" name="maxtaskCount">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Simultaneous downloads</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Default install</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QRadioButton" name="installmodeAttended">
            <property name="text">
             <string>Attended Install</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="installModeSilent">
            <property name="text">
             <string>Silent Install</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Proxy</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="proxy_settings">
         <property name="title">
          <string>Proxy Settings</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>HostName: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="proxy_hostname"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>         Port:  </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="proxy_port"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>         User: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="proxy_username"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string> Password: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="proxy_password"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>        Type: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="proxy_type">
              <item>
               <property name="text">
                <string>Socks5Proxy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HttpProxy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HttpCachingProxy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>FtpCachingProxy</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>48</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>checklatestversion</tabstop>
  <tabstop>checkstartwithsystem</tabstop>
  <tabstop>checkAskClose</tabstop>
  <tabstop>checkCloseClsoe</tabstop>
  <tabstop>checkCloseMinimize</tabstop>
  <tabstop>maxtaskCount</tabstop>
  <tabstop>installmodeAttended</tabstop>
  <tabstop>installModeSilent</tabstop>
  <tabstop>proxy_settings</tabstop>
  <tabstop>proxy_hostname</tabstop>
  <tabstop>proxy_port</tabstop>
  <tabstop>proxy_username</tabstop>
  <tabstop>proxy_password</tabstop>
  <tabstop>proxy_type</tabstop>
  <tabstop>btApply</tabstop>
  <tabstop>btCancel</tabstop>
  <tabstop>ckdlPackage</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
